generateData.js

```diff
@@ -6,7 +6,7 @@ const path = require("path")
 const fs = require("fs")
 
 const config = {
-    jsonMinify: true,
+    jsonMinify: true && !(process.argv.indexOf("-full") > -1),
```

Comment:
My logic senses tickle seeing this, true && is such a redundant statement :upside_down_face: